File: cloud_project/src/component/getJSON.js
const express = require("express"); //Line 1
const app = express(); //Line 2
const port = process.env.PORT || 5000; //Line 3

// This displays message that the server running and listening to specified port  https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/sqs-examples-send-receive-messages.html
// app.listen(port, () => console.log(`Listening on port ${port}`)); //Line 6

app.get("/json", (req, res) => {
  //   console.log("called");
  // res.send("YOUR EXPRESS BACKEND IS CONNECTED TO REACT"); //Line 10
  //   res.setHeader("Content-Type", "application/json");
  //   res.end(JSON.stringify({ a: 1 }, null, 3));
  res.header("Access-Control-Allow-Origin", "*");
  res.json({
    message: "Users retrieved",
    status: true,
    data: [
      {
        id: "0F8JIqi4zwvb77FGz6Wt",
        lastName: "RIDAM",
        firstName: "Heinz-Georg",
        email: "heinz-georg.fiedler@example.com",
        title: "mr",
        picture: "https://mjajaj.s3.amazonaws.com/download+(1).jpeg"
      },
      {
        id: "0P6E1d4nr0L1ntW8cjGU",
        picture: "https://mjajaj.s3.amazonaws.com/download+(2).jpeg",
        lastName: "Hughes",
        email: "katie.hughes@example.com",
        title: "miss",
        firstName: "Katie"
      },
      {
        id: "1Lkk06cOUCkiAsUXFLMN",
        title: "mr",
        lastName: "Aasland",
        firstName: "Vetle",
        picture: "https://mjajaj.s3.amazonaws.com/download+(3).jpeg",
        email: "vetle.aasland@example.com"
      },
      {
        id: "1OuR3CWOEsfISTpFxsG7",
        picture: "https://mjajaj.s3.amazonaws.com/download.jpeg",
        lastName: "Vasquez",
        email: "dylan.vasquez@example.com",
        title: "mr",
        firstName: "Dylan"
      },
      {
        id: "1pRsh5nXDIH3pjEOZ17A",
        lastName: "Vicente",
        title: "miss",
        firstName: "Margarita",
        email: "margarita.vicente@example.com",
        picture: "https://mjajaj.s3.amazonaws.com/images+(1).jpeg"
      }
    ]
  });
});

app.listen(7878);

File: getJson.py
import json


def lambda_handler(event, context):
    # TODO implement

    val = {
        'message': "Users retrieved",
        'status': '200'
    }
    print(val["message"])

    return {
        'message': "Users retrieved",
        'status': '200',
        'headers': {
            'Access-Control-Allow-Headers': 'Content-Type',
            'Access-Control-Allow-Origin': '*',
            'Access-Control-Allow-Methods': 'OPTIONS,POST,GET'
        },
        'data': [
            {
                'id': "1",
                'lastName': "Blue T-Shirt",
                'firstName': "",
                'picture': "https://getprodimgsrc.s3.amazonaws.com/download+(1).jpeg"
            }
            ,
            {
                'id': "2",
                'picture': "https://getprodimgsrc.s3.amazonaws.com/download+(2).jpeg",
                'lastName': "Grey Joggers",

                'firstName': ""
            },
            {
                'id': "3",

                'lastName': "Cream Joggers",

                'picture': "https://getprodimgsrc.s3.amazonaws.com/download+(3).jpeg",

            },
            {
                'id': "4",
                'picture': "https://getprodimgsrc.s3.amazonaws.com/download.jpeg",
                'lastName': "Navy blue T-Shirt",
                'email': "dylan.vasquez@example.com"

            },
            {
                'id': "5",
                'lastName': "Skin Color Joggers",

                'firstName': "",

                'picture': "https://getprodimgsrc.s3.amazonaws.com/images+(1).jpeg"
            },

            {
                'id': "6",
                'lastName': "Dalhousie Hoodies-Yellow",
                'firstName': "",

                'picture': "https://getprodimgsrc.s3.amazonaws.com/0001.jpg"
            },

            {
                'id': "7",
                'lastName': "DALHOUSIE Hoodie-Black",
                'firstName': "",

                'picture': "https://getprodimgsrc.s3.amazonaws.com/0001+(1).jpg"
            }
            ,
            {
                'id': "8",
                'lastName': "Dalhousie Hoodie White",
                'firstName': "",

                'picture': "https://getprodimgsrc.s3.amazonaws.com/0001+(2).jpg"
            },

            {
                'id': "9",
                'lastName': "Dalhousie Intramural hoodie",
                'firstName': "",

                'picture': "https://getprodimgsrc.s3.amazonaws.com/image_Dalhousie_hoodie.jpg"
            },
            {
                'id': "10",
                'lastName': "Nova Scotia Hoodie",
                'firstName': "",

                'picture': "https://getprodimgsrc.s3.amazonaws.com/s-l300.jpg"
            }
        ]
    }